src/arrays/multidimensional_arrays/hard/q3/CountsOdd.java

```diff
@@ -1,17 +1,30 @@
 package arrays.multidimensional_arrays.hard.q3;
 
 /*
-Make a Java program that defines a 3 by 3 multidimensional array in the main method and calls the countsOdd method. The
-countsOdd method should accept a multidimensional array of integers of any size, count the number of odd integers in the
-array, and print the count.
+Write a Java program that randomly initialize a 3 by 3 multidimensional array, with integers between 1 and 100
```

Comment:
This question needs to be changed more because it can't have a method like this, as this category is before arrays_with_methods